languages/en.ini

```diff
@@ -283,6 +283,7 @@ Description = "Description"
 Desired Username = "Desired Username"
 Detailed View = "Detailed View"
 Details = "Staff View"
+Digital = "Digital"
```

Comment:
Now that the Digital distinction has been removed, can we also remove these translations?

Reply:
Right, of course. Now removed.